blog/migrations/0001_initial.py

```diff
@@ -0,0 +1,49 @@
+"""Migration file."""
```

Comment:
Just exclude autogenerated files from pre-commit checks. There's no point in reformatting them.